src/templates/static/legal/metatrader/terms_and_conditions.jsx

```diff
@@ -1,446 +1,662 @@
+                    <li>{it.L('Whilst the Company endeavours to take all reasonable and sufficient steps, as required by applicable laws and regulations, to identify potential conflicts of interests between the Company and its Clients, or between one Client and another, that arise in the course of providing the Margin trading services as described in this agreement by establishing and implementing policies and procedures, it is possible that the Company may execute certain Orders which may have other direct or indirect material interests.')}</li>
+                    <li>{it.L('Given the Company\'s role as a Margin trading service provider as described in this agreement, the Company seeks to avoid undue market influence to the extent consistent with the Client\'s trading needs and the Company\'s risk management policies and procedures. By continuing to use the Margin trading services described in this agreement, the Client acknowledges that they are aware of the potential conflict of interest disclosed that may arise and cannot be completely eliminated, and they consent to the Company acting notwithstanding such potential conflict of interests.')}</li>
                 </ol>
             </li>
-            <li>{it.L('Client Money')}
+            <li>{it.L('Client money')}
                 <ol>
-                    <li>{it.L('Money collected from Clients are not invested in any securities, futures, or other investments, on behalf of Clients.')}</li>
-                    <li>{it.L('The company holds customer funds in separate bank accounts to the operational accounts; and arrangements have been made to ensure assets in the customer accounts are distributed to customers in the event of insolvency.')}</li>
-                    <li>{it.L('Your account should not be used as a banking facility and deposits should only be made with a view to using funds to place contracts. The Company is not a financial institution. Clients will not receive interest on deposits. Should you make repeated deposits and withdrawals without commensurate contracts being placed, we reserve the right to pass on to your accounts, without prior notice, any bank charges we have incurred before closing the account.')}</li>
+                    <li>{it.L('The money collected from Clients is not invested in any securities, futures, or other investments, on behalf of Clients.')}</li>
+                    <li>{it.L('The Company holds customer funds in bank accounts separate from the operational accounts, and arrangements have been made to ensure that the assets in the customer accounts will be distributed to the customers in the event of insolvency.')}</li>
+                    <li>{it.L('The Client\'s account should not be used as a banking facility, and deposits should only be made with a view to using funds for placing contracts. The Company is not a f
```

Comment:
`<li>{it.L('The [_1] website is protected by certain copyrights. The materials comprising the website (including, without limitation, all articles, text, images, logos, compilation, systems, code, and design) are protected by Copyright 1999-2018 Binary group of companies, all rights reserved. Such materials may be copied and distributed on a limited basis for non-commercial purposes only, provided that any material copied remain intact and that all copies include the following notice in a clearly visible position: "Copyright 1999-[_2] Binary Ltd. All rights reserved." These materials may not be copied or redistributed for commercial purposes or for compensation of any kind without prior written permission from a member of the Binary Group. [_1] and the bull/bear logo are registered trademarks.', it.website_name, '<span class=\'currentYear\'></span>')}</li>`